dataland-internal-storage/src/test/resources/db/migration/V18/originalFinancialCase2.json

```diff
@@ -0,0 +1,147 @@
+    "InsuranceOrReinsurance": {
+      "taxonomyEligibleActivityInPercent": null,
+      "taxonomyNonEligibleActivityInPercent": {
+        "value": 67.09,
+        "quality": "Incomplete",
+        "comment": "compress haptic circuit",
+        "dataSource": null
+      },
+      "derivativesInPercent": {
+        "value": null,
+        "comment": "navigate solid state monitor",
+        "dataSource": {
+          "page": 896,
+          "tagName": "interfaces",
+          "fileName": "IntegratedReport",
+          "fileReference": "10a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63"
+        }
+      },
+      "banksAndIssuersInPercent": {
+        "value": 49.41,
+        "quality": "Audited",
+        "comment": null,
+        "dataSource": {
+          "page": 1062,
+          "tagName": "relationships",
+          "fileName": "SustainabilityReport",
+          "fileReference": "20a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63"
+        }
+      },
+      "investmentNonNfrdInPercent": {
+        "value": 54.34,
+        "quality": "Incomplete",
+        "comment": "connect digital panel",
+        "dataSource": {
+          "page": 392,
+          "tagName": "systems",
+          "fileName": "IntegratedReport",
+          "fileReference": "10a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63"
+        }
+      }
+    }
+  },
+  "creditInstitutionKpis": null,
+  "investmentFirmKpis": null,
+  "insuranceKpis": {
+    "taxonomyEligibleNonLifeInsuranceActivitiesInPercent": {
+      "value": null,
+      "comment": "program optical pixel",
+      "dataSource": {
+        "page": 246,
+        "tagName": "technologies",
+        "fileName": "ESEFReport",
+        "fileReference": "30a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63"
+      }
+    }
+  },
+  "fiscalYearDeviation": "Deviation",
+  "fiscalYearEnd": "2023-12-14",
+  "scopeOfEntities": "No",
+  "nfrdMandatory": "No",
+  "euTaxonomyActivityLevelReporting": null,
+  "assurance": {
+    "value": "ReasonableAssurance",
+    "dataSource": {
+      "page": 733,
+      "tagName": "models",
+      "fileName": "IntegratedReport",
+      "fileReference": "10a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63"
+    },
+    "provider": "Gislason - Barton"
+  },
+  "numberOfEmployees": null,
+  "referencedReports": {
+    "ESEFReport": {
+      "fileReference": "30a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+      "isGroupLevel": null,
+      "reportDate": "2023-05-18",
+      "currency": "ISK"
+    },
+    "SustainabilityReport": {
+      "fileReference": "20a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+      "fileName": null,
+      "isGroupLevel": "No",
+      "reportDate": "2023-11-11",
+      "currency": "SYP"
+    },
+    "IntegratedReport": {
+      "fileReference": "10a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+      "fileName": "",
+      "isGroupLevel": null,
```

Comment:
aren't isGroupLevel and Currency removed from referenced reports? Makes me wonder if I forgot something in my older ticket. Or did you copy these fixtures from another migration test resource?

Reply:
You are right. The fields are not part of the data model. I removed them. I took the jsons from ./testing/data. They may be out of date.

Reply:
Ok. there was no need to remove them. I just wanted to make sure where they came from. all is fine now